### ISSUE TITLE
pvlist does not work with pvDatabase

### ISSUE THREAD

**Issue by mrkraimer** [epics-base/pvAccessCPP#132]
If I start
```
mrk> pwd
/home/epicsv4/masterCPP/exampleCPP/database/iocBoot/exampleDatabase
mrk> ../../bin/linux-x86_64/exampleDatabase st.cmd
...
epics> pvdbl
PVRBigRecord
PVRboolean
PVRbooleanArray

```
And in another window
```
mrk> pvlist
mrk> 
```
Nothing shows
In 7.0.1.1 this worked.

**Comment by mdavidsaver** [comment]
The likely culprit is 2d5ba1509528286fa9a75850c47c1263c1dfb691

**Comment by mdavidsaver** [comment]
Twas my doing.  Fix with  epics-base/pvDataCPP@e400d9f5fd28b522d84808bf748f0b713d38f30a

**Comment by mdavidsaver** [comment]
Released with Base 7.0.2
